jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -73,6 +76,9 @@ def __init__(self, *args, **kwargs):
 
         # subscribe to mouse events on any new viewers
         self.hub.subscribe(self, ViewerAddedMessage, handler=self._on_viewer_added)
+        self.hub.subscribe(
```

Comment:
Should we only subscribe when it is Cubeviz? Otherwise, all these other viz is going to listen to something that will never happen.

Reply:
@pllim, Thank you! How can we subscribe to specific configurations? I think adding filtering logic within the method for that particular configuration is all that is needed.
`if self.config == "cubeviz":` I am really curious to know this.

Reply:
Yes, similar to what happens a couple of lines above:

https://github.com/spacetelescope/jdaviz/blob/dc2186eb69da38b5ca8800777e6c16c1fdabf587/jdaviz/configs/imviz/plugins/coords_info/coords_info.py#L70

Reply:
And I can already hear Kyle's voice, "But what about free-form?" We can worry about that later because a lot more have to change, so this is just one more drop in the ocean.

Reply:
You could put the `self.hub.subscribe` call itself inside an `if self.config == "cubeviz"` check.

Reply:
this applies to any config with an image viewer though, right?  @gibsongreen has a ticket to enable unit conversion in all existing configs, which will hopefully get in for 4.0 if its feasible.